README.rst

```diff
@@ -75,8 +77,8 @@ in Python. Consider this example,
         value: Expression
 
 With such definition, a type checker can safely treat ``Node`` as
-``Union[Name, Operation, Assignment, Print]``, and also safely treat e.g.
-``Expression`` as ``Union[Name, Operation]``. So this will result in a type
+``Union[Expression, Statement]``, and also safely treat e.g.
+``Expression`` as ``Union[Name, Operation]`` and ``Statement`` as ``Union[Assignment, Print]``. So this will result in a type
```

Comment:
The PEP's I've looked at are all wrapped at 80 characters (as indicated by the `.editorconfig` in this project.

Let's use that convention as well so we match the existing documents.